### PR TITLE
Add ZK installation steps for RedHat

### PR DESCRIPTION
**Pull request by saptaktakalkar**

### CODE REVIEW

**Comment by abelboldu** [comment]
Vagrant test is failing:

`fatal: [centos]: FAILED! => {"changed": false, "failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'ansible_default_ipv4'"}`

File: defaults/main.yml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 ---
 # defaults file for zookeeper
+epel_repo_url: http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-6.noarch.rpm
```
Comment (abelboldu) [inline defaults/main.yml]: this is not needed.

`yum install epel-release`  should do

File: tasks/redhat.yml [hunk 2]
```diff
@@ -1,5 +1,21 @@
 ---
 
+- name: Install EPEL repo.
+  yum:
+    name: "{{ epel_repo_url }}"
+    state: present
```
Comment (abelboldu) [inline tasks/redhat.yml]: ditto

**Comment by saptaktakalkar** [comment]
`fatal: [centos]: FAILED! => {"changed": false, "failed": true, "msg": "AnsibleUndefinedVariable: 'dict object' has no attribute 'ansible_default_ipv4'"}`
@abelboldu This error is happening because `ansible_default_ipv4` is not accessible from one machine to another. So that's why I had said previously, to use only one box for testing, either `ubuntu` or `centos` and not both at once.

**Pull request merged**